Increase log level for dictionary loading

mlex opened the pull request:
So that the level of the 'Waiting for dictionaries to be loaded' message matches the level 'All dictionaries have been loaded'. This is especially nice when you load dictionaries on startup and only log INFO level.

<!---
A technical comment, you are free to remove or leave it as it is when PR is created
The following categories are used in the next scripts, update them accordingly
utils/changelog/changelog.py
tests/ci/cancel_and_rerun_workflow_lambda/app.py
-->

### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

davenger commented:
@mlex it looks like you mistakenly committed changes in many submodules while the real change is probably only this:
https://github.com/ClickHouse/ClickHouse/commit/0a0a5e41854f517ae2b3161c0526d1ae30f1ae01#diff-c7c4cea868f661341c1e9866836dc34c1c88723f9f33b4e09db530c2ea074036L3104-R3104

For me and some other colleagues github has hard time opening diff for this PR so maybe it submodules changes were not intentional it's probably better to re-create PR

mlex committed:
Increase log level for dictionary loading

So that the level of the 'Waiting for dictionaries to be loaded' message matches the level 'All dictionaries have been loaded'. This is especially nice when you load dictionaries on startup and only log INFO level.
